docs/widgets.md

```diff
@@ -39,6 +39,7 @@ Dashy has support for displaying dynamic content in the form of widgets. There a
   - [Mvg Departure](#mvg-departure)
   - [Mvg Connection](#mvg-connection)
   - [Custom search](#custom-search)
+  - [Rescuetime overview] (#rescuetime-overview)
```

Comment:
There's a space between the name and link which makes it not [render](https://github.com/Lissy93/dashy/blob/be0f51d46a3b0e57488f72faa43c4752c3a6cadb/docs/widgets.md) quite right.
```diff
- - [Rescuetime overview] (#rescuetime-overview)
                         ^
                         ╰ Just remove this ;)
+ - [Rescuetime overview](#rescuetime-overview)                       
```